src/commands/index.ts

```diff
@@ -25,6 +25,7 @@ import { utf8 } from './utf-8';
 import { calibrate } from './calibrate';
 import { CommandDefinition } from '../lib/command';
 import Logger from '../lib/logger';
+import { controls } from './controls';
```

Comment:
```suggestion
import { controls } from './controls';
import Logger from '../lib/logger';
```

Reply:
Can we move import { controls } etc. above import { CommandDefinition } etc. too?

It's nice to have the commands above other imports for clarity. Like in previous PR's